connect/config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
   // These variables are currently being used to retrieve above role members using API V3 `/roles` endpoint.
   // As soon as this endpoint is replaced with more suitable one, these variables has to be removed if no need anymore.
   CONNECT_MANAGER_ROLE_ID: 8,
+  CONNECT_COPILOT_MANAGER_ROLE_ID: 113,
```

Comment:
@gondzo I forgot how do we manage these ids when we test the application in dev env?

Reply:
We just use what is set in the admin app - role ids should be consistent between dev and prod